pyproject.toml

```diff
@@ -31,7 +31,6 @@ Issues = "https://github.com/bufbuild/protovalidate-python/issues"
 source = "vcs"
 
 [tool.ruff]
-target-version = "py38"
```

Comment:
https://docs.astral.sh/ruff/settings/#target-version — this will use the `requires-python` version if not set; don't need to keep the separate setting.